django/contrib/auth/management/commands/createsuperuser.py

```diff
@@ -219,11 +219,16 @@ def handle(self, *args, **options):
                 for field_name in self.UserModel.REQUIRED_FIELDS:
                     env_var = "DJANGO_SUPERUSER_" + field_name.upper()
                     value = options[field_name] or os.environ.get(env_var)
+                    field = self.UserModel._meta.get_field(field_name)
                     if not value:
+                        if field.blank and (
+                            options[field_name] is not None
+                            or os.environ.get(env_var) is not None
+                        ):
+                            continue
```

Comment:
I find this construction unusual, can we get rid of nested `if`?

Reply:
In short, I'm not so sure.
It's necessary to check if `value` is None and when it is, it requires additional checks for `field.blank` and if either of `options[field_name]`, `os.environ.get(env_var)` aren't None to determine if the command was used correctly.

I've tried to get rid of the nested `if` but I'm not convinced that the changed code is in any way better.

Or if you've meant moving `(options[field_name] is not None or os.environ.get(env_var) is not None`) to a variable and then using that variable in the `if` then sure.

Reply:
The only possible value is an empty string. IMO, it'd be clearer to use it.
```suggestion
                        if field.blank and (
                            options[field_name] == "" or os.environ.get(env_var) == ""
                        ):
                            continue
```